04-HTML5-APIs/indexedDB.js

```diff
@@ -0,0 +1,52 @@
+const DB_NAME = 'DB-texts';
+const DB_VERSION = 1; 
+const DB_STORE_NAME = 'texts';
+let db;
+let saveButton = document.getElementById('save');
+let clearButton=  document.getElementById('clear');
+
+
+function openDb() {
+    console.log("openDb ...");
+    var req = indexedDB.open(DB_NAME, DB_VERSION);
+    req.onsuccess = function (evt) {
+      db = this.result;
+      console.log("openDb DONE");
+    };
+    req.onerror = function (evt) {
+      console.error("openDb:", evt.target.errorCode);
+    };
+
+    req.onupgradeneeded = function (evt) {
+      console.log("openDb.onupgradeneeded");
+      evt.currentTarget.result.createObjectStore(
+        DB_STORE_NAME, { autoIncrement: false });
+
+    };
+  }
+
+
+saveButton.onclick = saveData;
+clearButton.onclick= clearData;
+
+
+function saveData(){
+    console.log('paso')
+    let data = document.getElementById('random-text').value
```

Comment:
aca deberias usar `const`

Reply:
**let**: The scope of a variable defined with the keyword “let” or “const” is limited to the “block” defined by curly braces i.e. {}.
“let” and “const” are“block scoped”.

**const**: The scope of a variable defined with the keyword “const” is limited to the block defined by curly braces. However if a variable is defined with keyword const, it cannot be reassigned.
“const” cannot be re-assigned to a new value. However it CAN be mutated.

https://codeburst.io/difference-between-var-let-and-const-in-javascript-fbce2fba7b4